presto-native-execution/presto_cpp/main/PrestoServer.h

```diff
@@ -232,6 +238,8 @@ class PrestoServer {
   // Executor for spilling.
   std::shared_ptr<folly::CPUThreadPoolExecutor> spillerExecutor_;
 
+  std::shared_ptr<facebook::presto::PrestoToVeloxPlanValidator> planValidator_;
```

Comment:
We probably need two APIs one is to init planValidator_ and the other is to get plan validator? The latter returns a raw pointer.